docs/release-process.md

```diff
@@ -1,5 +1,21 @@
 # PHP release process
 
+A release managers role includes making packaged source code from the canonical repository
+available according to their release schedule. The release schedule for their branch is
+available on a Wiki page.
+
+There are two types of release:
+
+  - non stable (alpha/beta/RC)
+  - stable
+  
+The process of making packaged source available and announcing availability is explained
+in detail below. The process differs slightly for non-stable and stable releases.
+
+Before a release manager can begin their work, there are several things that must happen,
+explained at the end of this document in 
+	[New Release Manager Checklist](#new-release-manager-checklist).
+
 ## General notes and tips
 
  1. Do not release on Fridays, Saturdays or Sundays because the sysadmins cannot
```

Comment:
I know this isn't your modified text, but "upgrade stuff" sounds super weak.

> 1. Do not release on Fridays, Saturdays, or Sundays as this gives poor lead time for downstream consumers adhering to a typical work week.

Reply:
Used your wording there, thanks.